lib/i18n/hygiene/checks/key_usage.rb

```diff
@@ -26,6 +26,12 @@ def run
             end
           end
         end
+
+        private
+
+        def thread_count
+          config.concurrency || Parallel.processor_count
+        end
```

Comment:
Is it worth having this fallback inside the config object so that anyone else deciding to use the `concurrency` option doesn't get caught out in the same way?